Browser example build error: Cannot find module 'vscode-languageserver/lib/protocol'.

Hello!

I am trying to re-build my theia image but some of the new features seems to have broken my build script and i would appreciate if someone has an idea of what might be wrong and how to fix it :)

I've got:
(Running in docker)
Ubuntu 16.04
Node 7.10
Npm 4.2 
Yarn 1.0.1

I'm running this in the docker build:
RUN useradd --create-home theia
USER theia
WORKDIR /home/theia
RUN git clone https://github.com/theia-ide/theia && cd theia **_&& yarn --unsafe-perm_**

It fails during the call to yarn, and gives the following error message:
**_theia/languages: [compile] src/node/language-server-contribution.ts(12,53): error TS2307: Cannot find module 'vscode-languageserver/lib/protocol'._**

Below is the short trace and i'll attach the "Full" trace.
Thank you in advance!
[FullTrace.txt](https://github.com/theia-ide/theia/files/1287361/TheiaBuildError.txt)


theia/preferences: > yarn run compile:clean && yarn run docs:clean && yarn run test:clean
lerna success run Ran npm script 'clean' in packages:
lerna success - theia/core
lerna success - theia/cpp
lerna success - theia/editor
lerna success - theia/filesystem
lerna success - theia/go
lerna success - theia/java
lerna success - theia/languages
lerna success - theia/monaco
lerna success - theia/navigator
lerna success - theia/preferences-api
lerna success - theia/preferences
lerna success - theia/python
lerna success - theia/terminal
lerna success - theia/workspace
lerna success - theia/example-browser
lerna success - theia/example-electron
lerna info version 2.0.0
lerna info scope theia/*
theia/core: > @theia/core@0.1.1 build /home/theia/theia/packages/core
theia/core: > concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"
theia/core: [lint] yarn run lint exited with code 0
theia/core: [compile] yarn run compile exited with code 0
theia/languages: > @theia/languages@0.1.1 build /home/theia/theia/packages/languages
theia/languages: > concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"
theia/preferences-api: > @theia/preferences-api@0.1.1 build /home/theia/theia/packages/preferences-api
theia/preferences-api: > concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"
theia/preferences-api: [lint] yarn run lint exited with code 0
theia/languages: [lint] yarn run lint exited with code 0
theia/preferences-api: [compile] yarn run compile exited with code 0
**_theia/languages: [compile] src/node/language-server-contribution.ts(12,53): error TS2307: Cannot find module 'vscode-languageserver/lib/protocol'._**
theia/languages: [compile] error Command failed with exit code 2.
theia/languages: [compile] yarn run compile exited with code 1
theia/languages: npm ERR! Linux 4.4.74-boot2docker
theia/languages: npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "build"
theia/languages: npm ERR! node v7.10.1
theia/languages: npm ERR! npm  v4.2.0
theia/languages: npm ERR! code ELIFECYCLE
theia/languages: npm ERR! errno 1
theia/languages: npm ERR! @theia/languages@0.1.1 build: `concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"`
theia/languages: npm ERR! Exit status 1
theia/languages: npm ERR!
theia/languages: npm ERR! Failed at the @theia/languages@0.1.1 build script 'concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"'.
theia/languages: npm ERR! Make sure you have the latest version of node.js and npm installed.
theia/languages: npm ERR! If you do, this is most likely a problem with the @theia/languages package,
theia/languages: npm ERR! not with npm itself.
theia/languages: npm ERR! Tell the author that this fails on your system:
theia/languages: npm ERR!     concurrently -n compile,lint -c blue,green "yarn run compile" "yarn run lint"
theia/languages: npm ERR! You can get information on how to open an issue for this project with:
theia/languages: npm ERR!     npm bugs @theia/languages
theia/languages: npm ERR! Or if that isn't available, you can get their info via:
theia/languages: npm ERR!     npm owner ls @theia/languages
theia/languages: npm ERR! There is likely additional logging output above.
theia/languages: npm ERR! Please include the following file with any support request:
theia/languages: npm ERR!     /home/theia/.npm/_logs/2017-09-08T08_50_42_781Z-debug.log
lerna ERR! execute Error: Command failed: npm run build
lerna ERR! execute
The command '/bin/sh -c git clone https://github.com/theia-ide/theia         && cd theia
       && yarn --unsafe-perm' returned a non-zero code: 1

--- a/packages/editor/extension.package.json
+++ b/packages/editor/extension.package.json
@@ -5,7 +5,7 @@
   "dependencies": {
     "@theia/core": "^0.1.1",
     "@theia/preferences": "^0.1.1",
-    "vscode-languageserver-types": "^3.3.0"
+    "vscode-languageserver-types": "^3.4.0"
   },
   "publishConfig": {
     "access": "public"

--- a/packages/languages/extension.package.json
+++ b/packages/languages/extension.package.json
@@ -5,7 +5,7 @@
   "dependencies": {
     "@theia/core": "^0.1.1",
     "vscode-base-languageclient": "^0.0.1-alpha.2",
-    "vscode-languageserver": "^3.3.0"
+    "vscode-languageserver": "^3.4.0"
   },
   "publishConfig": {
     "access": "public"

--- a/packages/languages/src/node/language-server-contribution.ts
+++ b/packages/languages/src/node/language-server-contribution.ts
@@ -9,7 +9,7 @@ import * as net from 'net';
 import * as cp from 'child_process';
 import { injectable } from "inversify";
 import { Message, isRequestMessage } from 'vscode-ws-jsonrpc';
-import { InitializeParams, InitializeRequest } from 'vscode-languageserver/lib/protocol';
+import { InitializeParams, InitializeRequest } from 'vscode-languageserver-protocol';
 import {
     createProcessSocketConnection,
     createProcessStreamConnection,

